internal/controller/controllers/crd_events_handler.go

```diff
@@ -31,9 +31,11 @@ func NewCRDEventsHandler() CRDEventsHandler {
 
 func (h *CRDEventsHandlerChannels) NotifyUpdates(ch chan<- event.GenericEvent, name string, namespace string) {
 	ch <- event.GenericEvent{
-		Meta: &metav1.ObjectMeta{
-			Name:      name,
-			Namespace: namespace,
+		Object: &metav1.PartialObjectMetadata{
```

Comment:
What's the significance of this change?

Reply:
`GenericEvent` was changed to take just take an `Object` (which already has the `Meta` info, rather than duplicating the `Meta` info. https://github.com/kubernetes-sigs/controller-runtime/commit/38d77dd0d4a4452cfc8bf898e7cff46cb3c91362#diff-f35b8976e8af7ef3dd90969c4b2b48f8cdb23e77e2a79a2be4fea7225938e13d

As part of that change, the type of `Object` was changed to `controllerutil` instead of `runtime.Object`. 

I'm not an expert in `controller-runtime` but using `PartialObjectMetadata` (which is basically the same as simply having `ObjectMeta` AFAIU) was the best way I found to preserve the current behavior.